README.md

```diff
@@ -3,9 +3,9 @@
 # ddev-matomo <!-- omit in toc -->
 
 
-## What is ddev-matomo?
+## What is DDEV Matomo?
```

Comment:
I prefer the old way.

Reply:
But this is from the style guide:

> 12. Update the `README.md` header in Title Case format, for example, use `# DDEV Redis`, not `# ddev-redis`.

https://github.com/ddev/ddev-addon-template?tab=readme-ov-file#getting-started

Reply:
See also https://github.com/ddev/ddev/issues/5641.

Reply:
The style guide got updated to use ddev-matomo.